spec/requests/users_spec.rb

```diff
@@ -1,7 +1,16 @@
 require 'rails_helper'
 
 RSpec.describe 'Users', type: :request do
-  describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+  describe 'GET #index' do
+    before(:example) { get '/users' } # get(:index)
+    it 'is a success' do
+      expect(response).to have_http_status(:ok)
+    end
+    it "renders 'index' template" do
+      expect(response).to render_template('index')
+    end
+    it 'includes correct placeholder text' do
+      expect(response.body).to include('Show users:')
+    end
```

Comment:
- Good job implementing the User `#index` action for this test 👍🏽. Kindly add a test for the User `#show` action to complete the test requirement 😉.